src/west/log.py

```diff
@@ -64,3 +71,42 @@ def die(*args, exit_code=1):
     print('fatal error:', end=' ', file=sys.stderr, flush=False)
     print(*args, file=sys.stderr)
     sys.exit(exit_code)
+
+
+def _print_color(*args, sep=' ', end='\n', file=sys.stdout, flush=False):
```

Comment:
I tested it and it looks nice enough on Windows.

Reply:
Did you test it with the particular ANSI escapes used here? Doesn't seem to support bright green (`\x1B[92m`) from looking at that page.

https://en.wikipedia.org/wiki/ANSI_escape_code lists that one as "aixterm (not in standard)" too, but some other things seem to use it. Looks a lot nicer than dark green.

Reply:
Wonder if it can wrap `stderr` too.

Reply:
Ah, yeah, looks like it does. Hadn't read far enough.

Reply:
It will remap the bright green to whatever:
![image](https://user-images.githubusercontent.com/12450381/45101031-e0058100-b12a-11e8-88de-2bc4fbfe39b7.png)

Reply:
Hmm... might get a brighter color with `\x1B[1m` (bold) too, though that'd probably give bold text on *nix.

Should be solvable at least. Maybe we could look into it after this gets in, to avoid too much stuff in a single PR.

Reply:
OK, I'll create an issue instead